kuma/static/js/wiki.js

```diff
@@ -679,4 +679,47 @@
         }
     }
 
+    // listens for post message from interactive editor
+    window.addEventListener('message', function(event) {
+        // get the interactive editor iframe
+        var iframe = document.querySelector('iframe.interactive');
+        var isExpectedOrigin = event.origin === 'https://interactive-examples.mdn.mozilla.net';
+
+        /* there may be other post messages so, ensure that the origin is the
+        expected and, that `event.data` contains an `iframeHeight` property */
+        if (isExpectedOrigin && event.data.iframeHeight) {
+            iframe.setAttribute('height', event.data.iframeHeight);
+        }
+    }, false);
+
+    // Insert notice about permanent language switch
+    var changedLocaleTo = localStorage.getItem('changed-locale-to');
+    if (changedLocaleTo) {
+        var locale = JSON.parse(changedLocaleTo),
+            text = gettext('You are now viewing this article in %(locale_name)s.' +
+                           ' Do you always want to view MDN articles in %(locale_name)s?'),
+            html = interpolate('<div id="locale-permanent-banner" class="primary overheadIndicator"><p>' + text +
+                               '<br><button id="locale-permanent-yes" data-locale="%(locale_code)s">' +
+                               gettext('Yes') + '</button> <button id="locale-permanent-no">' +
+                               gettext('No') + '</button></p></div>',
+                {locale_code: locale.code, locale_name: locale.name}, true);
+
+        $('#wiki-content').prepend(html);
```

Comment:
Please use the notification system for this. [The function is here.](https://github.com/mozilla/kuma/blob/7a7120f12c9349415877eacb040bfb393099d6d1/kuma/static/js/components.js#L473) I'm sorry we don't have any documentation on how to use it.  [Here's an example of how it's used when asking users to participate in our task completion survey](https://github.com/mozilla/kuma/blob/ad799cb8972c6b3857dc907cede8eddfbb0d5e86/kuma/static/js/task-completion.js#L36)e.